nemo_rl/distributed/batched_data_dict.py

```diff
@@ -514,6 +513,13 @@ def repeat_interleave(self, num_repeats: int) -> Self:
     def truncate_tensors(self, dim: int, truncated_len: int):
         """Truncates tensors in this dict of a given dim to a given length."""
         for k, v in self.items():
+            # keys not to truncate (anything other than input_ids, attention_mask, etc.)
+            # @rohitkumarj;; TODO: make this configurable later
+            if k in ['image_grid_thw', 'pixel_values']:
```

Comment:
How do we get these keys? Model arch specific?

Reply:
Found a better way to handle this by extracting the list of strings from the `vlm_keys` argument